src/test/unit_tests/test_nc_nsfs_cli.js

```diff
@@ -378,10 +396,20 @@ mocha.describe('manage_nsfs cli', function() {
             assert_account(account, account_options);
         });
 
+        mocha.it('cli account status without name and access_key', async function() {
```

Comment:
Should be written using jest as decided a few weeks ago. adding to mocha is not something that we want
@nimrod-becker FYI